imaging.rst

```diff
@@ -64,6 +64,39 @@ QR Code® is a registered trademark of DENSO WAVE INCORPORATED.
 
 ----
 
+.. raw:: latex
+
+    \clearpage
+
+.. _1c7d74:
+.. index:: $1C $7D $74 - Set 2D Barcode Size
+
+.. py:attribute:: Set 2D Barcode Size - $1C $7D $74 k
+
+   Sets the width/height of QR code cells, measured in dots (8 dots = 1mm). 
+
+   .. note:: Requires firmware 1.29 or newer
+
+   :Format:
+       ``Hex       $1C  $7D $74 k``  
+
+       ``ASCII     FS   }   t   k``  
+        
+       ``Decimal   28  125  116 k``  
+   :Notes:
+       - If the QR code generated with the configured size setting is too big to be printed on the paper being used, the printer will automatically choose smaller sizes, down to the smallest valid size (3 dots per cell) until a suitable size has been found. If no suitable size can be found, the QR code will not be printed. This will only occur when using paper smaller than 80mm with the largest valid size.
```

Comment:
Can you add a note about setting the paper size since that is independent from the ESC/POS API?